docs/contribution.md

```diff
@@ -0,0 +1,13 @@
+Please be aware that we are not actively looking for feature contributions. The truth is that it's extremely difficult for someone outside SonarSource to comply with our roadmap and expectations. Therefore, we typically only accept minor cosmetic changes and typo fixes. If you would like to see a new feature, please create a new thread in the forum ["Suggest new features"](https://community.sonarsource.com/c/suggestions/features).
+
+With that in mind, if you would like to submit a code contribution, make sure that you adhere to the following guidelines and all tests are passing (Travis build is executed for each pull request):
```

Comment:
nitpick: I think it should be Cirrus build not Travis

Reply:
Good point